drivers/include/hdc1000.h

```diff
@@ -138,6 +138,22 @@ int hdc1000_trigger_conversion(const hdc1000_t *dev);
  */
 int hdc1000_get_results(const hdc1000_t *dev, int16_t *temp, int16_t *hum);
 
+/**
+ * @brief   Convenience function for reading temperature and humidity
```

Comment:
IMHO this it not a 'convenience' function, but it extends the base read function with new functionality (i.e. caching). So the comment is misleading.